be/src/cloud/cloud_engine_calc_delete_bitmap_task.cpp

```diff
@@ -203,7 +200,6 @@ Status CloudTabletCalcDeleteBitmapTask::handle() const {
         }
         auto error_st =
                 Status::Error<ErrorCode::DELETE_BITMAP_LOCK_ERROR, false>("version not continuous");
-        _engine_calc_delete_bitmap_task->add_error_tablet_id(_tablet_id, error_st);
         return error_st;
```

Comment:
return Status::Error<ErrorCode::DELETE_BITMAP_LOCK_ERROR, false>("version not continuous");